.nuspec/Microsoft.Maui.Resizetizer.targets

```diff
@@ -339,34 +339,39 @@
             Input="@(MauiAsset)">
             <Output ItemName="AndroidAsset"          TaskParameter="Output" Condition="'$(_ResizetizerIsAndroidApp)' == 'True'" />
             <Output ItemName="Content"               TaskParameter="Output" Condition="'$(_ResizetizerIsiOSApp)' == 'True'" />
-            <Output ItemName="ContentWithTargetPath" TaskParameter="Output" Condition="'$(_ResizetizerIsUWPApp)' == 'True' Or '$(_ResizetizerIsWindowsAppSdk)' == 'True'" />
+            <Output ItemName="ContentWithTargetPath" TaskParameter="Output" Condition="'$(_ResizetizerIsWindowsAppSdk)' == 'True'" />
             <Output ItemName="TizenResource" TaskParameter="Output" Condition="'$(_ResizetizerIsTizenApp)' == 'True'" />
         </GetMauiAssetPath>
     </Target>
 
     <Target Name="ProcessMauiSplashScreens"
-        Condition="'@(MauiSplashScreen)' != ''"
         Inputs="$(MSBuildThisFileFullPath);$(_ResizetizerTaskAssemblyName);$(_MauiSplashInputsFile);@(MauiSplashScreen)"
         Outputs="$(_MauiSplashStampFile)">
 
+        <PropertyGroup>
+            <_MauiHasSplashScreens>false</_MauiHasSplashScreens>
+            <_MauiHasSplashScreens Condition="'@(MauiSplashScreen->Count())' != '0'">true</_MauiHasSplashScreens>
+        </PropertyGroup>
```

Comment:
The is partially the fix. Since apps may need a splash in order to launch right, we actually always want to process a splash and create an empty one if none was found. For now, the other platforms have "ok" fallbacks - windows/maccat do not support splash, android just used the primary color and tizen needs to be tested still. 

However, iOS always requires one so this PR makes this target always run, we just skip the steps for the other platforms. When building for iOS, we run the tasks as if we had a splash, and when the actual task is running it determines no splash was provided so generates an empty one.

This may also be nice if we detect no splash on Android, we can just use the app icon. But for now, this PR is for iOS.